### PR TITLE
PDA-184 - bulk action disabled when no rows selected

### PR DESCRIPTION
**Pull request by rimselis**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/TaskList/TaskListContainer.js
Comment (jeffgo10) [inline widgets/src/components/TaskList/TaskListContainer.js]: why are the permissions removed?

**Review by jeffgo10 — COMMENTED** [review]
just one question

**Comment by entando-service** [comment]
All is well

**Review by jeffgo10 — APPROVED** [review]
cool

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/101#pullrequestreview-392064170" title="Approved">jeffgo10</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
